sim/validator.py

```diff
@@ -116,6 +116,34 @@ def pipeline(
         shutil.rmtree(btb_seq_backup_path)
         shutil.rmtree(results_path)
 
+def benchmark(genomes_path, results_path, output_path):
+    # Validate input
+    if not os.path.exists(results_path):
```

Comment:
same as below, wondering if it would be better to have this in `sequenced.from_directory`?

Reply:
agree done